src/noshow/api/app.py

```diff
@@ -119,10 +119,18 @@ async def predict(
     project_path = Path(__file__).parents[3]
     start_time = datetime.now()
 
+    if len(input) == 0:
+        logger.error("No appointments provided")
```

Comment:
I think the cleaner option would be to raise an exception, because generally we want to be informed when the appointment list is empty, this is only expected a couple of times per year. For example:
```raise HTTPException(status_code=400, detail="Input cannot be empty.")```
 maybe best to do both logger.error and the exception